mesheryctl/pkg/utils/utils_test.go

```diff
@@ -369,153 +363,3 @@ spec:
-			version:      "master",
-		},
-	}
-	for _, tt := range tests {
-		t.Run(tt.name, func(t *testing.T) {
-			// Expected response
-			testdataDir := filepath.Join(currDir, "testdata")
-			golden := NewGoldenFile(t, tt.expectOutput, testdataDir)
-			// mock response
-			apiResponse := NewGoldenFile(t, tt.fixture, fixturesDir).Load()
-			httpmock.RegisterResponder("GET", tt.url,
-				httpmock.NewStringResponder(200, apiResponse))
-			actualurl, err := GetManifestTreeURL(tt.version)
-			if err != nil {
-				if tt.expectErr {
-					if *update {
-						golden.Write(err.Error())
-					}
-					expectedResponse := golden.Load()
-					Equals(t, expectedResponse, err.Error())
-					return
-				}
-			}
-			if *update {
-				golden.Write(actualurl)
-			}
-			expectedResponse := golden.Load()
-			Equals(t, expectedResponse, actualurl)
-		})
-	}
-	// stop mock server
-	StopMockery(t)
-}
-
-func TestListManifests(t *testing.T) {
-	// initialize mock server for handling requests
-	StartMockery(t)
-
-	// get current directory
-	_, filename, _, ok := runtime.Caller(0)
-	if !ok {
-		t.Fatal("Not able to get current working directory")
-	}
-	currDir := filepath.Dir(filename)
-	fixturesDir := filepath.Join(currDir, "fixtures")
-	tests := []struct {
-		name         string
-		url          string
-		expectOutput string
-		fixture      string
-		expectErr    bool
-	}{
-		{
-			name:         "Test listing manifests",
-			expectErr:    false,
-			expectOutput: "listmanifest.expect.golden",
-			fixture:      "listmanifest.api.response.golden",
-			url:          "https://api.github.com/repos/meshery/meshery/git/trees/47c634a49e6d143a54d734437a26ad233146ddf5",
-		},
-		{
-			name:         "Test listing manifests with wrong url",
-			expectErr:    true,
-			expectOutput: "listmanifest.expect.err.golden",
-			fixture:      "listmanifest.api.err.response.golden",
-			url:          "https://api.github.com/repos/meshery/meshery/git/trees/gibberish",
-		},
-	}
-	for _, tt := range tests {
-		t.Run(tt.name, func(t *testing.T) {
-			// Expected response
-			testdataDir := filepath.Join(currDir, "testdata")
-			golden := NewGoldenFile(t, tt.expectOutput, testdataDir)
-			// mock response
-			apiResponse := NewGoldenFile(t, tt.fixture, fixturesDir).Load()
-			httpmock.RegisterResponder("GET", tt.url,
-				httpmock.NewStringResponder(200, apiResponse))
-			manifests, err := ListManifests(tt.url)
-			if err != nil {
-				if tt.expectErr {
-					if *update {
-						golden.Write(err.Error())
-					}
-					expectedResponse := golden.Load()
-					Equals(t, expectedResponse, err.Error())
-					return
-				}
-			}
-			manifestsactual, err := json.Marshal(&manifests)
-			if err != nil {
-				t.Error("Could not unmarshall manifests from response")
-			}
-			if *update {
-				golden.Write(string(manifestsactual))
-			}
-			expectedResponse := golden.Load()
-			Equals(t, expectedResponse, string(manifestsactual))
-		})
-	}
-	// stop mock server
-	StopMockery(t)
-}
```

Comment:
Any reason why you are removing these tests? I'm writing the tests for `helpers.go` and some functions are covered here. I guess I can move these tests inside the dedicated file where I'm writing other tests. :+1:

Reply:
Yup that's what I did too, I moved these two tests to the dedicated platform.go test file. I didn't delete them :)